pkg/internal/api/common.go

```diff
@@ -133,3 +136,28 @@ func (c *ClientImpl) doRequest(ctx context.Context, req *http.Request) ([]byte,
 
 	return body, err
 }
+
+func (c *ClientImpl) runQuery(ctx context.Context, serviceID string, sql string, args ...interface{}) ([]byte, error) {
+	qry, err := sqlbuilder.ClickHouse.Interpolate(sql, args)
+	if err != nil {
+		return nil, err
+	}
+
+	s := struct {
+		Sql string `json:"sql"`
```

Comment:
```suggestion
		SQL string `json:"sql"`
```

nit: practice is acronyms in names in Go should be the either full lower or full case (depend whether we expose outside package or not).